test/server/cards/02_SHD/upgrades/Ruthlessness.spec.ts

```diff
@@ -0,0 +1,80 @@
+describe('Ruthlessness', function() {
+    integration(function(contextRef) {
+        describe('Ruthlessness\'s ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: [{ card: 'battlefield-marine', upgrades: ['ruthlessness'] }],
+                    },
+                    player2: {
+                        groundArena: ['atst']
+                    }
+                });
+            });
+
+            it('should not deal 2 damage to base when not defeating a enemy unit', function () {
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.battlefieldMarine);
+                context.player1.clickCard(context.atst);
+
+                expect(context.battlefieldMarine.location).toBe('discard');
+                expect(context.p2Base.damage).toBe(0);
+                expect(context.player2).toBeActivePlayer();
+            });
+        });
+
+        describe('Ruthlessness\'s ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: [{ card: 'battlefield-marine', upgrades: ['ruthlessness'] }],
+                    },
+                    player2: {
+                        groundArena: ['specforce-soldier']
+                    }
+                });
+            });
+
+            it('should deal 2 damage to base when defeating a enemy unit (and survives)', function () {
```

Comment:
We should add the same set of tests that Mace has since the ability is basically the same. There's a couple missing here